Add player info doc to website (#102)
Irreverent-DevOps/irreverentdevops-web: src/pages/player-info.md

DevOps Party Games takes this idea and tilts it on its head by adding DevOps-inspired content to the Jackbox games [Drawful 2](https://www.jackboxgames.com/drawful-two/) and [Quiplash 2](https://www.jackboxgames.com/quiplash-two-interlashional/), and then streams it live via [Twitch](https://twitch.tv/mattstratton) for a worldwide audience to watch, comment, and hopefully be entertained. In addition, the hosts ([Matt Stratton](https://twitter.com/mattstratton), [Jeremy Meiss](https://twitter.com/IAmJerdog), and [Dan Maher](https://twitter.com/phrawzty)) will provide color commentary, much like a modern day Cotton McKnight and Pepper Brooks (announcers from Dodgeball). The goal here is to have fun and not take things too seriously.

If you’d like to see an example, previous episodes can be viewed at [http://matty.wtf/devops-party-games](http://matty.wtf/devops-party-games).

# The Format

> There will be two rounds - one round of Quiplash, and one round of Drawful.

Reviewer: ```suggestion
There are three rounds: one each of Quiplash, Thriplash, and Drawful.
```

Author: hmm. technically thriplash (from the player perspective) is just part of the quiplash game. not sure if this will add confusion, but I also have no strong objection.

Author: the reason that I don't say you separate them is that they are scheduled as one thing, and also they are ranked together for TLB

There will be a maximum of 8 contestants, who will play both rounds.

We will be using “custom episodes” of Drawful and Quiplash, where all the prompts will be related to devops/cloud computing/etc. 

So for example, a prompt you might get in Drawful is that you need to draw a picture of “full stack engineer” and the other players have to try to guess that!

## Code of Conduct

> All players will need to abide by a [Code of Conduct](https://devopspartygames.com/code-of-conduct). Remember that this will be broadcast live via Twitch, and also will be recorded to be viewed later, so keep that in mind!

Reviewer: ```suggestion
All players must abide by a [Code of Conduct](https://devopspartygames.com/code-of-conduct). Remember that this will be both broadcast live via Twitch and recorded for Youtube—so keep that in mind!
```

> Unofficially, the ESRB rating for this will be “M” (Mature, 17+)

Reviewer: ```suggestion
Unofficially, our ESRB rating is “M” (Mature, 17+).
```

That said, please refrain from sexualized jokes/content in gameplay.

## Logistics
All contestants and hosts will connect to Zoom video call (the Zoom link will be in your calendar invite). Your video will be part of the stream—if you would prefer to be “camera off” please just let us know.

> Players will see the “screen” of the game via Zoom screen sharing. They will use a second device (tablet, smartphone, etc) as the “controller” for the game (the place where they provide input/draw pictures, etc).

Reviewer: ```suggestion
Players will see the “screen” of the game via Zoom screen sharing. They will use a second device (tablet, smartphone, etc) as the “controller” for the game (the place where they provide input/draw pictures, etc). Players with multiple displays (or one _really big_ one) can use the same screen for both.
```

Author: yes, but we might want to remind them that the drawing games can be hard on a computer vs mobile device?

> The game will be streamed to [Matty’s Twitch channel](https://www.twitch.tv/mattstratton/), which is how the “audience” can watch. The Twitch channel will display the main screen of the game, plus the cameras of all players and hosts. All audio from the game (plus the Zoom call) will also be streamed to Twitch, so that contestants will be able to be heard.

Reviewer: ```suggestion
The game will be streamed to [Matty’s Twitch channel](https://www.twitch.tv/mattstratton/), which is how the audience watches and participates (more on that later). The Twitch channel displays the main screen of the game, plus the cameras of all players and hosts. All audio from the game (plus the Zoom call) is also streamed to Twitch, and audience interaction is encouraged!
```

> People watching the game can join the “audience” of the game on their personal devices; this lets them vote/participate in the rankings for the game as well! The audience doesn’t “play the game” but they vote on the players’ responses and can potentially influence the winners!

Reviewer: ```suggestion
The audience can join the game on their personal devices. While they don't play the games directly, they can vote on answers, and this can have a big impact on scoring. (Yes, pandering to the audience is absolutely a valid strategy.)
```

> There will be 6-8 players, who will play both rounds. The hosts (Matty, Jeremy, Dan) will not be playing (it would be slightly unfair since we wrote the prompts!)

Reviewer: ```suggestion
There will be 6–8 players, who will play both rounds. The hosts don't play (it would be slightly unfair since we know the prompts ahead of time).
```

## Discord
If you are not already in our Discord server, please join with the invite link at [https://devopspartygames.com/discord](https://devopspartygames.com/discord). You will need to accept the Code of Conduct to get access to the server. **Please let an admin know when you have joined, and we will grant you the “Players” role that gives you more access in Discord.** We will use Discord for coordination and chatting amongst each other prior to the game, etc.

Note that the Discord is public for members of the community; there are private channels just for players and moderators though!

## Schedule

> Episodes are streamed at **20:00 CET / Paris** (EMEA league) or **20:00 CT / Chicago** (Americas league) on a **Tuesday**!  The game itself (all rounds) will likely take about an hour, but we will schedule an extra 30 minutes prior for “green room” time to make sure everything is working as expected. Please join the Discord voice channel (“Game”) **30 minutes before show time on game night** so we can do tech checks, etc.
> 
> You will receive a calendar invite for the game, which is the time you should join the "Game" channel in Discord and be ready to go.

Reviewer: ```suggestion
Episodes are generally streamed at **20:00 CET / Paris** (EMEA league) or **20:00 CT / Chicago** (Americas league) on a **Tuesday**. The game itself (all rounds) takes about an hour, but we schedule an extra 30 minutes prior for “green room” time to make sure everything is working as expected. Please join the Zoom call **30 minutes before show time on game night** so we can do tech checks, etc.

You will receive a calendar invite for the game, which will have all the salient details you'll need to connect.
```

We will also provide the ability for a “practice” session (optional) which will be “classic” Drawful and Quiplash, which could be helpful for players who are new to these games. You will get a calendar invite for this practice game; it’s totally optional, but join if you want! It should be fun.

> Here is a rough schedule (all times are the time zone of the game)
> 
> 19:30: Players join Discord and do audio checks/tech checks, etc<br />
> 19:50: Stream on twitch.tv/mattstratton starts with countdown to show timer<br />

Reviewer: ```suggestion
Here's the rough schedule:

19:30: Players join Discord and do audio checks/tech checks, etc.<br />
19:50: Stream on twitch.tv/mattstratton starts with “countdown to show” timer<br />
```

20:00: Show starts streaming live, audio from players is “hot”, etc. Hosts introduce the game and introduce the players<br />
20:05: Start playing first game<br />
20:30: Start playing second game<br />
20:55: End of show; thank everyone for playing/watching, etc. 

Note that these times are estimated; we aren’t 100% sure how long each round will take, etc, so it’s POSSIBLE the show might go a few minutes over.

## FAQ
*When will this happen?*<br />
**EMEA League** Episodes usually start at 20:00 CET / Paris on the second Tuesday of each month.<br />
**Americas League** Episodes usually start at 20:00 CT / Chicago on the last Tuesday of each month.

Also, see the **Schedule** section directly above.

*Do I need to install anything?*<br>

> You do not need to install anything to play the games. However, you need to have Zoom installed on a computer, and also a mobile device with a web browser (tablet or smartphone).

Reviewer: ```suggestion
You need to have a functional Zoom client installed on a device, and also a modern web browser to play the games.
```

> You do not need to install anything to play the games. However, you need to have Zoom installed on a computer, and also a mobile device with a web browser (tablet or smartphone).
> 
> You can install Discord via [https://discord.com/download](https://discord.com/download) or if you use [Homebrew](https://brew.sh), just
> <br>
> `brew cask install discord`

Reviewer: ```suggestion
You need to have a functional Zoom client (capable of both audio and video) to join the stream, and a relatively modern web browser to play the games.
```

If you’d like to see an example, previous episodes can be viewed at [http://matty.wtf/devops-party-games](http://matty.wtf/devops-party-games).

# The Format
There will be two rounds - one round of Quiplash, and one round of Drawful.

There will be a maximum of 8 contestants, who will play both rounds.

We will be using “custom episodes” of Drawful and Quiplash, where all the prompts will be related to devops/cloud computing/etc. 

So for example, a prompt you might get in Drawful is that you need to draw a picture of “full stack engineer” and the other players have to try to guess that!

## Code of Conduct
All players will need to abide by a [Code of Conduct](https://devopspartygames.com/code-of-conduct). Remember that this will be broadcast live via Twitch, and also will be recorded to be viewed later, so keep that in mind!

Unofficially, the ESRB rating for this will be “M” (Mature, 17+)

That said, please refrain from sexualized jokes/content in gameplay.

## Logistics

> All contestants will connect to a private voice room in Discord (your video will be part of the stream; if you would prefer to be “camera off” please just let us know) - please note that you will need to use the Discord app, not just the web interface, for this. The invite to Discord is included in the calendar invitation you should have received. If you need a new one, email games@devopspartygames.com.

Reviewer: ```suggestion
All contestants and hosts will connect to a private voice room in Discord. Your video will be part of the stream—if you would prefer to be “camera off” please just let us know. Note that you will need to use the Discord app (not just the web interface) for this, as A/V in the web interface is wonky. The invite to Discord is included in the calendar invitation you should have received, but if you need a new one just email us at games@devopspartygames.com.
```

Author: we actually need to update this with the general Discord invite, since we don't have a Players specific one anymore.

Players will see the “screen” of the game via Discord screen sharing. They will use a second device (tablet, smartphone, etc) as the “controller” for the game (the place where they provide input/draw pictures, etc).

> The game will be streamed to [Matty’s Twitch channel](https://www.twitch.tv/mattstratton/), which is how the “audience” can watch. The Twitch channel will display the main screen of the game, plus indicators of who is in the Discord room and when they are speaking. All audio from the game (plus the Discord call) will also be streamed to Twitch, so that contestants will be able to be heard.

Reviewer: Z-Z-Z-Z-Zoooom

```suggestion
All contestants and hosts will connect to a Zoom session. Your video will be part of the stream—if you would prefer to be “camera off” please just let us know. Everybody will see the “screen” of the game via Zoom screen sharing. You'll use a second device such as a tablet or smartphone as the “controller” for the game. If you have two displays (or one _really_ big one) you can use the same computer for both.

The game will be streamed to [Matty’s Twitch channel](https://www.twitch.tv/mattstratton/), which is how the “audience” can watch. The Twitch channel will display the main screen of the game, plus audio and video from the Zoom session, and player/audience interaction is encouraged!
```

Author: udpated with this

> Please note: your camera **will** be on for the stream, so be “camera ready” (whatever that means to you!). If you would prefer to NOT have your camera show in the stream, please let Matty know.

Reviewer: This is covered above.
```suggestion
```

People watching the game can join the “audience” of the game on their personal devices; this lets them vote/participate in the rankings for the game as well! The audience doesn’t “play the game” but they vote on the players’ responses and can potentially influence the winners!

There will be 6-8 players, who will play both rounds. The hosts (Matty, Jeremy, Dan) will not be playing (it would be slightly unfair since we wrote the prompts!)

## Discord

> If you are not already in our Discord server, please join with the invite link at [https://devopspartygames.com/discord](https://devopspartygames.com/discord). You will need to accept the Code of Conduct to get access to the server. **Please let an admin know when you have joined, and we will grant you the “Players” role that gives you more access in Discord.** We will be capturing/streaming the audio from the Discord chat to the livestream. Also, bear in mind that your Discord username and avatar will be displayed on the stream (it will “light up” when you are speaking; see the screenshot below for an example)
> 
> ![](/images/discord-overlay-example.jpg)
> 
> Once you join Discord, send a note anyone listed under Admins in the Discord (Matty, Jeremy, or Dan) and let us know you are in; we will grant you the proper role to give you access to the “Game” voice channel (and the “players” text channel). The “Game” voice channel is what you will need to join/connect audio to during the actual stream/event. The text channel is just in case we want it. 
> 
> You also should set your “nickname” on our Discord server to your actual name - that is what will display in the overlay instead of your Discord username. It will just look nicer :)
> 
> Note that the Discord is public for members of the community; there are private channels just for players and moderators though!

Reviewer: Most of this is moot once we move to Zoom.

```suggestion
If you aren't already in our Discord server, please join with the invite link at [https://devopspartygames.com/discord](https://devopspartygames.com/discord). You will need to accept the Code of Conduct to get access to the server.

**Please let an admin know when you have joined, and we will grant you the “Players” role that gives you more access in Discord.**

Note that the Discord is public for members of the community; there are private channels just for players and moderators though!
```

Note that the Discord is public for members of the community; there are private channels just for players and moderators though!

## Schedule

> Episodes are streamed at **20:00 UTC+1** (EMEA league) or **20:00 UTC-6** (Americas league) on a **Tuesday**!  The game itself (all rounds) will likely take about an hour, but we will schedule an extra 30 minutes prior for “green room” time to make sure everything is working as expected. Please join the Discord voice channel (“Game”) **30 minutes before show time on game night** so we can do tech checks, etc.

Reviewer: Those UTC markers are valid for standard time only; summer time shifts them both by +1.

```suggestion
Episodes are streamed at **20:00 CET / Paris** (EMEA league) or **20:00 CT / Chicago** (Americas league) on a **Tuesday**!  The game itself (all rounds) will likely take about an hour, but we will schedule an extra 30 minutes prior for “green room” time to make sure everything is working as expected. Please join the Discord voice channel (“Game”) **30 minutes before show time on game night** so we can do tech checks, etc.
```

You will receive a calendar invite for the game, which is the time you should join the "Game" channel in Discord and be ready to go.

We will also provide the ability for a “practice” session (optional) which will be “classic” Drawful and Quiplash, which could be helpful for players who are new to these games. You will get a calendar invite for this practice game; it’s totally optional, but join if you want! It should be fun.

Here is a rough schedule (all times are the time zone of the game)

19:30: Players join Discord and do audio checks/tech checks, etc<br />
19:50: Stream on twitch.tv/mattstratton starts with countdown to show timer<br />
20:00: Show starts streaming live, audio from players is “hot”, etc. Hosts introduce the game and introduce the players<br />
20:05: Start playing first game<br />
20:30: Start playing second game<br />
20:55: End of show; thank everyone for playing/watching, etc. 

Note that these times are estimated; we aren’t 100% sure how long each round will take, etc, so it’s POSSIBLE the show might go a few minutes over.

## FAQ
*When will this happen?*<br />

> **EMEA League** - 
> Episodes stream at 20:00 UTC+1 on the second Tuesday of each month.<br />
> **Americas League** Episodes stream at 20:00 UTC-6 on the last Tuesday of each month.

Reviewer: Those UTC markers are valid for standard time only; summer time shifts them both by +1.

```suggestion
**EMEA League** Episodes stream at 20:00 CET / Paris on the second Tuesday of each month.<br />
**Americas League** Episodes stream at 20:00 CT / Chicago on the last Tuesday of each month.
```
